web/landing_page.html

```diff
@@ -30,6 +30,14 @@ <h1>{{.Name}}</h1>
       </div>
       {{ end }}
       {{ .ExtraHTML }}
+      <div id="pprof">
+      Go pprof debugging profile data:
+      <ul>
+        <li><a href="debug/pprof/heap">heap usage (memory)</a>
+        <li><a href="debug/pprof/profile">CPU usage (takes some time to gather)</a>
```

Comment:
You could add `?seconds=10` to make it less alarming for the newcomer.

Reply:
Yes, but that has a high chance of missing a scrape interval. I was actually thinking about making this `60s` so we gather a few scrapes. Or capture instances where people reduce the scrape interval.